packages/visual-editor/src/components/Dropzone/EditorBlock.tsx

```diff
@@ -94,6 +94,42 @@ export const EditorBlock: React.FC<EditorBlockProps> = ({
     });
   };
 
+  if (isComponentMissing || !definition || !elementToRender) {
+    const renderMissingComponentPlacehoder = () => (
+      <div
+        style={{
+          border: '1px solid red',
+          width: '100%',
+          height: '100%',
+        }}>
+        Missing component &apos;{node.data.blockId}&apos;
+      </div>
+    );
```

Comment:
Since this same component is defined in two files, this could potentially be moved to its own file and imported into both places it's used.

Reply:
✅  FIXED 65ad695c0ba70b01158c1492491aaa947127b931